php/laravel/security/laravel-active-debug-code.php

```diff
@@ -0,0 +1,23 @@
+<?php
+    // ruleid: laravel-active-debug-code 
+    config(['app.debug' => 'true']);
+
+    // ruleid: laravel-active-debug-code 
+    putenv("APP_DEBUG=true");
+
+    // ruleid: laravel-active-debug-code 
+    $_ENV['APP_DEBUG'] = 'true';
+
+    // ok: laravel-active-debug-code
+    config(['app.debug' => 'false']);
+
+    // ok: laravel-active-debug-code
+    putenv("APP_DEBUG=false");
+
+    // ok: laravel-active-debug-code 
+    $_ENV['APP_DEBUG'] = 'false';
+
+    // this is ok because it retrieves the value from the env file instead of setting it directly.
+    // ok: laravel-active-debug-code
+    $value = config('app.debug', 'true');
```

Comment:
`true` is a default value here - should we still flag it? Failing to set it in the environment will cause this to "fail open" to debug behavior.

Reply:
weird, totally thought that I'd replied but perhaps I didn't click send 🤔 . this is definitely a good question -- I was thinking about this while writing the rule, and thought that we're currently prioritizing reducing FP rather than FN so opted to not catch unsafe defaults. Do you think we'll get a lot of negative sentiment from developers due to this?

Reply:
This is a good point. The defaults is a "maybe problem" and perhaps we're okay with this?  We can ship this as-is and revisit if we get lots of "why didn't you find this" reports.